OpenSim/Simulation/Model/AbstractPathPoint.cpp

```diff
@@ -75,10 +75,14 @@ void AbstractPathPoint::updateFromXMLNode(SimTK::Xml::Element& aNode,
             if (bodyElement != aNode.element_end()) {
                 bodyElement->getValueAs<std::string>(bodyName);
                 // PathPoints in pre-4.0 models are necessarily 3 levels deep
-                // (model, muscle, geometry path), and Bodies are necessarily
-                // 1 level deep: prepend "../../../" to get the correct
-                // relative path.
-                if (!bodyName.empty()) bodyName = "../../../" + bodyName;
+                // (model, muscle, geometry path), and Bodies are
+                // necessarily 1 levels deep; here we create the correct
+                // relative path (accounting for sets being components).
+                if (bodyName == "ground") {
+                    bodyName = "../../../../" + bodyName;
+                } else if (!bodyName.empty()) {
+                    bodyName = "../../../../bodyset/" + bodyName;
+                }
```

Comment:
@chrisdembia was that leaked over from an earlier change? Without this PR I can till open the tug-of-war model at 3.3 format and pathpoints on ground are properly attached, any clues?